dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/HibernateTrackedEntityStore.java

```diff
@@ -659,10 +683,12 @@ private String getDescendantsQuery(TrackedEntityQueryParams params) {
     orgUnits.append("AND (");
 
     for (OrganisationUnit organisationUnit : params.getOrgUnits()) {
-
-      OrganisationUnit ou = organisationUnitStore.getByUid(organisationUnit.getUid());
-      if (ou != null) {
-        orgUnits.append(orHlp.or()).append("OU.path LIKE '").append(ou.getPath()).append("%'");
+      if (organisationUnit != null) {
```

Comment:
Confused about this not null check. The previous code fetched something from db before checking null